app/pubspec.yaml

```diff
@@ -94,7 +94,11 @@ dependencies:
   path: ^1.8.3
   url_launcher: ^6.1.10
   file_picker: ^8.0.3
-  emoji_picker_flutter: ^2.1.1
+  emoji_picker_flutter: #^3.0.0
+    # path: ../../emoji_picker_flutter
+    git:
+      url: https://github.com/acterglobal/emoji_picker_flutter
+      ref: main-no-web
```

Comment:
Can I quick summary of changes in this library?

Reply:
I removed that change after. See https://github.com/acterglobal/a3/pull/2108 for the final result. Once merged, this will still just be using the released version.